lametro/management/commands/compile_pdfs.py

```diff
@@ -177,58 +173,53 @@ def findEventAgendaPacket(self, all_documents=False):
                 ON i.bill_id=d_bill.bill_id
                 INNER JOIN councilmatic_core_eventdocument as d_event
                 ON i.event_id=d_event.event_id
-                INNER JOIN councilmatic_core_bill AS b
-                ON d_bill.bill_id=b.ocd_id
+        '''
+
+        if all_documents:
+            query += '''
```

Comment:
I find this type of thing really hard to follow. please write out both queries even if they are very similar.